src/test/scala/mesosphere/marathon/integration/AppDeployIntegrationTest.scala

```diff
@@ -396,6 +396,29 @@ class AppDeployIntegrationTest
     pingTask(taskUpdate2).entityString should be(s"Pong $appId\n")
   }
 
+  test("stop (focefully delete) a deployment") {
+    Given("a new app that is not healthy")
+    val appId = testBasePath / "failing"
+    val app = appProxy(appId, "v1", instances = 1, withHealth = false).
+      copy(healthChecks = Set(HealthCheck(gracePeriod = 20.second, interval = 1.second, maxConsecutiveFailures = 10)))
+    appProxyCheck(appId, "v1", state = false)
+    val create = marathon.createApp(app)
+    create.code should be (201) // Created
+    val deploymentId = extractDeploymentIds(create).head
+
+    Then("the deployment can not be finished")
+    intercept[AssertionError] {
+      waitForEvent("deployment_success")
```

Comment:
We always have to wait 30 seconds for this. Is that really necessary? Should id not be enough to check for the (still) running deployment?